eva-pipeline/src/main/resources/application.properties

```diff
@@ -1 +1,55 @@
-spring.profiles.active=production
+#JOBs
+spring.profiles.active=test
+spring.batch.job.names=variantJob
+#spring.batch.job.names=variantLoadJob
+
+#LOG
+#https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-logging.html
+#debug=true
+#logging.level.=INFO
+logging.level.embl.ebi.variation.eva=DEBUG
+logging.level.org.opencb.opencga=DEBUG
+logging.level.org.springframework=INFO
+
+
+input=/Users/diego/projects/EVA/biodata/biodata-formats/src/test/resources/variant-test-file.vcf
+fileId=5
+aggregated=NONE
+studyType=COLLECTION
+studyName=diego
+studyId=7
+outputDir=
+pedigree=
+dbName=variants
+storageEngine=mongodb
+compressGenotypes=true
+compressExtension=.gz
+includeSrc=FIRST_8_COLUMNS
+
+skipLoad=false
+opencga.app.home=/Users/diego/projects/EVA/opencga/opencga-app/build
+
+#VEP
+vepInput=/path/to/variants_to_annotate.tsv.gz
+vepOutput=/path/to/variants_annotation.tsv.gz
+vepPath=
+vepCacheDirectory=
+vepCacheVersion=
+vepSpecies=
+vepFasta=
+vepNumForks=
+
+
+overwriteStats=false
+calculateStats=false
+includeSamples=false
+annotate=false
+includeStats=false
+
+#SKIP STEPs
+#skipLoad
+#skipStatsCreate
+#skipStatsLoad
+skipAnnotGenerateInput=true
+skipAnnotCreate=true
+skipAnnotLoad=true
```

Comment:
Yes the most common scenario should be in the property file... this one is just random to be able to test the process